tools/importer/import.js

```diff
@@ -115,7 +117,7 @@ export default {
       prevOffset = currentOffset;
       offsetDiff = block.querySelectorAll('div').length + 1;
     });
-
+  
```

Comment:
Remove these extra spaces added.
Not needed and format the code before raising the PR.